flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/AggregateFunctionDefinition.java

```diff
@@ -23,10 +23,8 @@
 import org.apache.flink.table.functions.UserDefinedAggregateFunction;
 import org.apache.flink.util.Preconditions;
 
-import static org.apache.flink.table.expressions.FunctionDefinition.Type.AGGREGATE_FUNCTION;
-
 /**
- * The function definition of an user-defined aggregate function.
+ * The function definition of an user-defined aggregate or a table aggregate function.
  */
 @PublicEvolving
 public final class AggregateFunctionDefinition extends FunctionDefinition {
```

Comment:
Can we also introduce a new type of function definition? The type should not depend on the implementation itself. In the future, the implementation might not be available.